org-formation/650-identity-providers/_tasks.yaml

```diff
@@ -166,6 +166,30 @@ GithubOidcSageBionetworksSynapseDockerRegistry:
       - !Ref SynapseProdAccount
     Region: us-east-1
 
+GithubOidcSageBionetworksSynapse:
```

Comment:
do you plan to also replace the role used by GH actions in `synapse-docker-registry` repo with the one generated from this new `GithubOidcSageBionetworksSynapse` resource?  And then of course also remove `GithubOidcSageBionetworksSynapseDockerRegistry`?

Reply:
Correct.